label_studio/tests/utils.py

```diff
@@ -221,6 +221,35 @@ def redis_client_mock():
         yield
 
 
+@contextmanager
+def openai_client_mock():
+    import openai
+    import requests
+
+    from openai import AuthenticationError
+
+    class DummyOpenAIModels:
```

Comment:
Is it worth using the library Nikolai suggested for this? (Or another one like it). I figure we may need to mock more functionality like this as we depend on more features of the api.
On the other hand, I think the obvious next thing is listing models which is included here, so I think I'm good with whatever your sense for this is.

Reply:
Yeah I think doing it ourselves in this case is the smoothest path to go down